### ISSUE TITLE
Intellisense not behaving properly before class object is instantiated

### ISSUE THREAD

**Issue by fflaten** [PowerShell/PowerShellEditorServices#1896]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the latest version
- [X] Search the existing issues.

### Steps to reproduce

```powershell
class Person {
    [string]$Name
    [int]$Age = 999

    Person([string]$name) {
        $this.Name = $name
    }

    Person([string]$name, [int]$age) {
        $this.Name = $name
        $this.Age = $age
    }

    [string]GetIntroduction([string]$greeting) {
        return "$greeting $($this.Name)"
    }

    [string]GetIntroduction([bool]$withAge) {
        return "Hello $($this.Name) ($($this.Age))"
    }
}

# 1. Run until this point. Don't execute the next lines
# ctrl+space at end $p. to open intellisense.
# Look at GetIntroduction (missing second overload) and Person (shouldn't be there)
$p = [Person]::new("Frode")
$p.

# 2. Execute next line
$p = [Person]::new("Frode")

# 3. ctrl+space at end of next line and check again. both method overloads visible + no constructor as expected
$p.
```

### Expected behavior

```console
Intellisense/completion shows the same step 3 at step 2
```

### Actual behavior

```console
Difference. Intellisense for members of `$p` include constructor and only first overload of methods. See images below
```

### Environment data

```PowerShell
Name                           Value
----                           -----
PSVersion                      7.2.5
PSEdition                      Core
GitCommitId                    7.2.5
OS                             Linux 5.10.102.1-microsoft-standard-WSL2 #1 SMP Wed Mar 2 00:30:59 UTC 2022
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Version

2022.8.2 preview

### Visuals

Constructor is shown
![image](https://user-images.githubusercontent.com/3436158/184972426-4a1a3a26-1c61-4122-bd60-f8326bf02268.png)

Only first overload
![image](https://user-images.githubusercontent.com/3436158/184973258-ea74d9ac-3334-45e1-81df-b039d3046b7f.png)

**Comment by MartinGC94** [comment]
PowerShell provides the tooltips, not editor services. This issue was fixed in PS version 7.3 preview 5, specifically this PR: https://github.com/PowerShell/PowerShell/pull/16963 so if you can run a preview version there's a fix available, otherwise you will need to wait for the final release.

**Comment by fflaten** [comment]
@MartinGC94 You Sir, are one step ahead of the rest.🥇  Thanks!
